features/contract-state/Totals.tsx

```diff
@@ -87,10 +89,29 @@ const Totals = () => {
     prettyTotalTokens: string = defaultMissingDataDisplay,
     prettyCollSymbol: string = "",
     prettyTokenSymbol: string = "",
-    getExchangeLinkCollateral: string = "https://app.uniswap.org/#/swap",
-    getExchangeLinkToken: string = "https://app.uniswap.org/#/swap",
+    getExchangeLinkCollateral: string = "",
+    getExchangeLinkToken: string = "",
     exchangeName: string = "Uniswap"
   ) {
+    const addTokenToMetamask = async () => {
+      // Add token to users metamask wallet.
+      if (provider == null) return;
+
+      // @ts-ignore
+      provider.send("wallet_watchAsset", {
+        // @ts-ignore
+        type: "ERC20",
+        options: {
+          address: tokenAddress,
+          symbol: tokenSymbol?.substring(0, 6),
```

Comment:
If you put this within the `if` statement above you will not need to have the `?` after `tokenSymbol` because you will already have checked for that

Reply:
great, good catch